packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -100,6 +134,12 @@ export class Stepper {
   connectedCallback(): void {
     this.mutationObserver?.observe(this.el, { childList: true });
```

Comment:
unrelated : this observer is never disconnected. Is this intentional ?

Reply:
Should be disconnected.

Reply:
Added https://github.com/Esri/calcite-design-system/pull/8013